Gordon360/Controllers/RecIM/ParticipantsController.cs

```diff
@@ -65,6 +64,15 @@ public async Task<ActionResult> AddParticipant(int participantID)
             return (Ok(participantID));
         }
 
+        [HttpPost]
+        [Route("notifications/{username}")]
```

Comment:
I would reverse this to:
```suggestion
        [Route("{username}/notifications")]
```
When you are posting a new object, you generally specify the last part of the URL as the object type. So if you are posting to notifications, you would want to post to `.../notifications` not `.../notifications/{username}`. Get what I mean?

Reply:
Yes, I was not putting too much thought into the specific route name at the time but that does make sense